x-pack/solutions/search/plugins/enterprise_search/common/constants.ts

```diff
@@ -208,6 +208,7 @@ export const MANAGE_API_KEYS_URL = '/app/management/security/api_keys';
 
 export const ENTERPRISE_SEARCH_DOCUMENTS_DEFAULT_DOC_COUNT = 25;
 
+// TODO: Remove?
```

Comment:
we can remove `ENTERPRISE_SEARCH_CONNECTOR_CRAWLER_SERVICE_TYPE` since we won't have it in 9.x

Reply:
I'll remove this in my next PR since it's also used in the server some. But those usages get removed in my follow-up

Reply:
This is still used a lot in the UI, those can be removed but going to punt that as it's not as important as other things that need updating for now.